pkg/application/options.go

```diff
@@ -243,8 +248,14 @@ func WithLoggerSettingsFromConfig(app *App) {
 		settings := &loggerSettings{}
 		config.UnmarshalKey("mon.logger", settings)
 
+		outputFile, err := os.OpenFile(settings.Output.File, os.O_CREATE|os.O_WRONLY, 0755)
```

Comment:
Why can't you pass the logger output file as an additional option when creating your application?

E.g. `app.New(application.WithLoggerOutput(os.Stdout))`

I don't see a benefit of requiring an output file for any kind of application.

Reply:
@selm0 anything is a file in unix/linux/mac world. Even stdout/stderr are.

I do not want to have to add it as an option, because it makes sense to have it as a logging option instead.
Usually you can configure applications, where they should log to.
Consider any kind of linux service.

Reply:
Having a file output configurable doesn't reveal on how it's implemented.

I still think it should be up to the application to define, where they want to log to.
With your addition of the new option stack to the cli, it's even easier to use `WithLoggerOutput`.

Assume I want to pass my logs via a channel to a consumer, then I'd still be stuck with a file being opened unnecessarily.

Reply:
> Having a file output configurable doesn't reveal on how it's implemented.

Why would it need to reveal to the "user" how it is implemented? Nothing changes in how the log is written, so the requirement shouldn't change within this PR imho.

> I still think it should be up to the application to define, where they want to log to.

This is not how one would usually write applications.
The user/administrator of the pc executing it is in charge to configure it, if they don't like the defaults.
Or do you mean that there should just be the default os.StdOut via an application option, but still my config flag?

> Assume I want to pass my logs via a channel to a consumer, then I'd still be stuck with a file being opened unnecessarily.

Totally new scenario here that we never talked about.
With a channel do you mean a generic communication channel as in a connection or are we talking about go channels here?
If you mean a generic communication channel, then this is what my next PR is actually about: Logging to an external destination via tcp or udp.